packages/core/components/AutoField/fields/SelectField/index.tsx

```diff
@@ -27,18 +28,20 @@ export const SelectField = ({
     >
       <select
         id={id}
+        title={label || name}
         className={getClassName("input")}
         disabled={readOnly}
-        onChange={(e) => {
+        onChange={({ target: { value } }) => {
```

Comment:
I guess this is a deliberate switch from `currentTarget` to `target`? Makes sense